{{cookiecutter.project_slug}}/clients/vue3/src/App.vue

```diff
@@ -37,33 +57,8 @@ export default {
 }
 </script>
 
-<style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
-
-#nav {
-  padding: 30px;
-}
-
-#nav a {
-  font-weight: bold;
-  color: #2c3e50;
-}
-
-#nav #logout {
-  /* Same style as a link */
-  font-weight: bold;
-  color: #2c3e50;
-  text-decoration: underline;
-  cursor: pointer;
-}
-
-#nav a.router-link-exact-active {
-  color: #42b983;
+<style scoped lang="css">
+.router-link-exact-active {
+  color: theme('colors.accent');
```

Comment:
**Info:** Kept the styling for the active router links at the scoped level in this `App` view` to match the previous set up, but this styling could get moved to a global level if there's a desire to keep the active router link styles consistent app-wide.

Reply:
in router > index.js 

in createRouter you can add the following:

createRouter({
history: createWebHistory(),
routes,
**linkActiveClass: 'active-link'**,
**linkExactActiveClass: 'exact-active-link'**
})

Then in index.css you can add global styles (or do it in an individual component if need be) and do something like:

.exact-active-link {
@apply text-primary;
}